src/main/kotlin/com/workos/webhooks/models/UserUpdated.kt

```diff
@@ -79,5 +74,65 @@ open class UserUpdated
    */
   @JvmField
   @JsonProperty("previous_attributes")
-  val previousAttributes: Map<String, Any?>,
-) : User(obj, id, directoryId, organizationId, idpId, userName, firstName, lastName, jobTitle, createdAt, updatedAt, emails, groups, state, customAttributes, rawAttributes)
+  val previousAttributes: Map<String, Any?>
+) : PreviousAttributes(obj, id, directoryId, organizationId, idpId, userName, firstName, lastName, jobTitle, createdAt, updatedAt, emails, state, customAttributes, rawAttributes)
```

Comment:
I think we should be able to drop this entirely (and the `PreviousAttributes` class)

```suggestion
)
```

Reply:
Updated